client/src/views/hoverView.js

```diff
@@ -27,8 +25,8 @@ function createPlanetStats(planet){
   return ul;
 };
 
-var HoverView = function(planet){
+var Hover = function(planet){
   return createHoverDiv(planet);
 };
 
-module.exports = HoverView;
+module.exports = Hover;
```

Comment:
Why have you done this rather than `module.exports = createHoverDiv`?

Reply:
not sure I was following it all through and must have got confused, I changed it back